src/components/box/_box.scss

```diff
@@ -0,0 +1,14 @@
+$boxBorderColor: $graySecondary;
+
+.mint-box {
+  @include component;
+
+  border-radius: 22px;
```

Comment:
Asia told me that `22px` and `10px` are only two valid values for `border-radius`. I don't think there any correlation with `defaultLayoutPadding`.